src/main/asciidoc/groovy/index.adoc

```diff
@@ -485,6 +485,26 @@ Base name: `vertx.datagram`
 * `<host>:<port>.bytes-read` - A <<histogram>> of the number of bytes read.
 ** This metric will only be available if the datagram socket is listening
 
+=== Pool metrics
+
+Base name: `vertx.pool.<type>.<name>` where `type` is the type of the pool (e.g _worker_, _datasource_) and
+`name` is the name of the pool (e.g `vert.x-worker-thread`).
+
+Pools of type _worker_ are blocking worker pools. Vert.x exposes its worker as _vert.x-worker-thread_ and
+_vert.x-internal-blocking_. Named worker executor created with `link:../../groovydoc/io/vertx/groovy/core/WorkerExecutor.html[WorkerExecutor]` are exposed.
```

Comment:
Would be nice to add a sentence about the purpose of each of these internal pools. That would help people in charge of vertx app in production but not necessarily aware of vertx implementation details.